.github/workflows/support-window.yml

```diff
@@ -1,52 +1,49 @@
 name: Update support window diagram
 on:
-  # When our dependencies change
+  # When our dependencies change.
   push:
     paths:
       - .github/workflows/support-window.yml
       - package.json
-      - scripts/support-window.ts
-      - scripts/tsconfig.json
-  # Manually, when TypeScript is released
+      - scripts/support-window.js
+      - scripts/jsconfig.json
+  # Manually, when TypeScript is released.
   # https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow
   workflow_dispatch:
-permissions:
-  contents: read
```

Comment:
I would probably leave this, because it ensures any added jobs need to be explicitly changed to allow writing. At least, that's what it appears to have been intended to do.

Reply:
@jakebailey Can we change the DT organization's default workflow permissions from <kbd>Read and write</kbd> to <kbd>Read repository contents</kbd>? https://docs.github.com/en/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization#configuring-the-default-github_token-permissions

That's the new default:
> By default, when you create a new organization, GITHUB_TOKEN only has read access for the contents scope.

- https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61065#issue-1292109424 made it safe to make that change for the DT repo, by setting any additional permissions in all of the workflows.
- The [dt-mergebot](https://github.com/DefinitelyTyped/dt-mergebot/tree/master/.github/workflows) and [focus-dt](https://github.com/DefinitelyTyped/focus-dt/tree/master/.github/workflows) repos don't use GITHUB_TOKEN/automatic token authentication, except for actions/checkout, which is fine with <kbd>Read repository contents permission</kbd>.
- The DT organization's [other repos](https://github.com/orgs/DefinitelyTyped/repositories) don't use GitHub Actions (except for [dts-critic](https://github.com/DefinitelyTyped/dts-critic/tree/master/.github/workflows), which is archived/read-only).

Reply:
That's a good idea, I had forgotten that they added that option. I don't have permission, but maybe @RyanCavanaugh or @andrewbranch or @sandersn do?

Reply:
I can do this, but don’t feel like I’m 100% confident in what the effects will be. Not going to do it on a Friday 😄 I can look next week with Nathan if we feel like it’s important.

Reply:
Thanks!

Reply:
Done.

Reply:
Thanks!